fluentd/README.md

```diff
@@ -52,7 +52,63 @@ In your fluentd configuration file, add a `monitor_agent` source:
 
 #### Log Collection
 
-Follow [those instructions][4] to forward logs to Datadog with Fluentd.
+As long as you can forward your FluentD logs over tcp/udp to a specific port, you can use that approach to forward your FluentD logs to your Datadog agent. But another option is to use the [Datadog FluentD plugin][10] to forward the logs directly from FluentD to your Datadog account. 
+
+##### Add metadata to your logs
+
+In order to get the best use out of your logs in Datadog, it is important to have the proper metadata associated with your logs (including hostname and source). By default, the hostname and timestamp should be properly remapped thanks to our default [remapping for reserved attributes][12]. 
```

Comment:
comma needed here maybe `remapped, thanks`, maybe?

Reply:
Proper metadata (including hostname and source) is the key to unlocking the full potential of your logs in Datadog. By default, the hostname and timestamp fields should be properly remapped via [remapping for reserved attributes][12].

?

Reply:
=> they should contact support then :)